blockbase/block-template-parts/footer.html

```diff
@@ -0,0 +1,3 @@
+<!-- wp:paragraph {"align":"center","fontSize":"small"} -->
+<p class="has-text-align-center has-small-font-size">Proudly powered by <a href="https://wordpress.org">WordPress</a></p>
```

Comment:
Just out of curiosity, what's the benefit of naming the classes `has-text-align-center` and `has-small-font-size` over  `text-align-center` and `font-size-small`?

Reply:
That all comes straight from Gutenberg. 🙂

Reply:
Ah, I see. Thanks for clarifying this, @kjellr. 🙌